src/Table/Column/Tooltip.php

```diff
@@ -72,7 +72,7 @@ public function getHtmlTags(Model $row, ?Field $field): array
 
         return [
             '_' . $field->shortName . '_data_visible_class' => '',
-            '_' . $field->shortName . '_data_tooltip' => $tooltip,
+            '_' . $field->shortName . '_data_tooltip' => $this->getApp()->uiPersistence->typecastSaveField($row->getModel()->getField($this->tooltipField), $tooltip),
```

Comment:
how is this PR different to https://github.com/atk4/ui/pull/2076?

Reply:
You're right - we don't move along with bug-fixing, so issues are identified now again and again. Sorry. Would be nice to have this primitive bug fixed asap, so duplication in issue reporting and PR submission won't happen again.

Reply:
closing in favor of the original PR